ARK Hardware Docs

patrikpordi opened the pull request:
<!--

### Solved Problem
ARK Harware were outdated

### Solution
Added the missing ones and edited the existing ones slightly

-->

mrpollo commented:
Hey @patrikpordi can you please remove the submodule updates so reviews can be cleaner? Thanks!

patrikpordi committed:
Add ARK documentation for GPS modules, distance sensors, and power modules

- Added DAN, SAM, SAM Mini, X20, RTK L1 L5 GPS modules
- Added ARK distance sensors documentation
- Added ARK power modules documentation

hamishwillee commented on docs/en/power_module/ark_12s_payload_power_module.md:
```diff
@@ -0,0 +1,59 @@
+# ARK 12S Payload Power Module
```
How does this differ from https://docs.px4.io/main/en/power_module/ark_12s_pab_power_module

hamishwillee replied:
Specifically I mean the names are a little similar and readers might not differentiate their purpose.

hamishwillee replied:
@patrikpordi Absolutely worth having separately. My point is that the reason for this  "connector to power additional devices" is not obvious, so it isn't clear why you would chose this. 

I've added a clarification in https://github.com/PX4/PX4-Autopilot/pull/25991/commits/b1a3039af5e7b5d27c324554a1b27d5f2bc0bc6b 

> This is similar to the [ARK 12S PAB Power Module](../power_module/ark_12s_pab_power_module.md) except that the additional 12V 6A supply allows easier powering of a payload.

hamishwillee committed:
pm no connector - prettier and fix heading

hamishwillee committed:
Shrink all images

hamishwillee committed:
Add to sidebar except no connector

hamishwillee committed:
Prettier and minor fixes

hamishwillee reviewed (commented):
This looks great thanks @patrikpordi - awful lot to review though, so I'll have another look after you've responded to my comments.

patrikpordi commented:
@hamishwillee Thank you!

patrikpordi commented on docs/en/power_module/ark_pab_power_module_no_connector.md:
```diff
@@ -0,0 +1,47 @@
+# ARK PAB Power Module No Connector
```
This might be better just to be included in the ARK PAB Power Module as an option.

hamishwillee replied:
I think so, if it has no other differences. I'll leave that to you

hamishwillee commented on docs/en/power_module/index.md:
```diff
@@ -26,7 +26,9 @@ This section provides information about a number of power modules and power dist
   - [Sky-Drones SmartAP PDB](../power_module/sky-drones_smartap-pdb.md)
 - Digital (I2C) voltage and current power modules (for Pixhawk FMUv6X and FMUv5X derived controllers):
   - [ARK PAB Power Module](../power_module/ark_pab_power_module.md)
+  - [ARK PAB Power Module No Connector](../power_module/ark_pab_power_module_no_connector.md)
```
Yes, please remove.

patrikpordi committed:
Removed Power Module No connector version, included it in Power Module.

hamishwillee committed:
Power supply tweaks

hamishwillee commented on docs/en/power_module/ark_pab_power_module.md:
```diff
@@ -6,35 +6,36 @@ Note that at 60A and 20°C without cooling, the 5V regulator is de-rated to a 3A
 
 ![ARK PAB Power Module](../../assets/hardware/power_module/ark_power_modules//ark_pab_power_module.jpg)
 
+This power module is also available without connectors:
```
FYI, I just moved the text a bit. Makes it more clear what you're looking at.

hamishwillee committed:
ark_rtk_gps_l1_l2 - numbering hints

hamishwillee reviewed (approved):
@patrikpordi Thanks very much for the changes. As commented I made some tweaks for you to look at. I am approving, and will merge tomorrow
